### ISSUE TITLE
missed klines

### ISSUE THREAD

**Issue by BoogalooLi** [binance/binance-public-data#262]
i checked the data and find missed klines
[missing.csv](https://github.com/binance/binance-public-data/files/12414070/missing.csv)

**Comment by crabbit22** [comment]
yes please -- can someone please fix? I presume some routine job stopped working.

@BoogalooLi do you have an alternate source for the klines? I need 1m eth-usd data

**Comment by crabbit22** [comment]
They fixed it. I raised an issue with binance folks via chat :)

**Comment by 2pd** [comment]
Sorry I'm not very sure I can understand the file format in the "missing.csv" file.
Can you give a few example of which file, and the timestamp of the missing data?

thank you

**Comment by crabbit22** [comment]
@2pd Not sure about the missing file from @BoogalooLi but i noticed missing 1m klines for Oct, Nov and Dec for ETHUDC in 2022 -https://data.binance.vision/?prefix=data/spot/daily/klines/ETHUSDC/1m/

Any chance these could be backfilled?

**Comment by 2pd** [comment]
@crabbit22  can you give me an example of the missing kline in any of the months?

thanks

**Comment by utkarshp** [comment]
Having the same issue as @crabbit22. Klines is missing data ETH/USDC minutely data from Sep 30, 2022 through Mar 11, 2023. I also think the files for Sep 29, 2022 and Mar 12, 2023 are suspiciously small. For example, https://data.binance.vision/data/spot/daily/klines/ETHUSDC/1m/ETHUSDC-1m-2022-09-28.zip gives a file of size 53KB, whereas https://data.binance.vision/data/spot/daily/klines/ETHUSDC/1m/ETHUSDC-1m-2022-09-29.zip gives a file of size 5.4 KB. After that, https://data.binance.vision/data/spot/daily/klines/ETHUSDC/1m/ETHUSDC-1m-2022-09-30.zip gives a 404 not found error.

**Comment by crabbit22** [comment]
Yep, an alternative that's free (you get 50k credits) but won't be binance orderbook is from transpose on weth token. You can resample it to be by min

**Comment by Realsmoovlyke** [comment]
How do I recover an old Bitcoin mine
